src/client/app/containers/MapChartContainer.ts

```diff
@@ -88,18 +158,23 @@ function mapStateToProps(state: State) {
 				marker: {
 					color: colors,
 					opacity: 0.5,
-					size
+					size,
+					sizemin: 6,
+					sizeref: scaling,
+					sizemode: 'area'
 				},
 				text: texts,
 				opacity: 1,
 				showlegend: false
 			};
 			data.push(traceOne);
+			// }
 		}
 	}
 
 	// set map background image
 	const layout: any = {
+		// Either the actual map name or text to say it is not available.
 		title: {
 			text: (map) ? map.name : 'There\'s not an available map'
```

Comment:
I'm not sure if this text ('There\\'s not an available map') is displayed in the actual UI anywhere, but it might have to be internationalized?

Reply:
Issue #574 will address this as there are a number of places that maps do not use internationalized strings.